extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -563,11 +590,13 @@ protected function getSearchConfigFromRequest() {
 			$requestedFields = $searchConfig->getRequestedFields();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$requestedFields" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)